Makefile

```diff
@@ -288,23 +296,27 @@ dist_manifest_in_files := \
     README.md \
     README_PYPI.md \
     requirements.txt \
-    *.py \
+    pyproject.toml \
     $(package_name)/*.py \
     $(package_name)/*/*.py \
     $(package_name)/*/*/*.py \
 
 # Files that are dependents of the distribution archive.
 # Keep in sync with dist_manifest_in_files.
-dist_dependent_files := \
+dist_dependent_files_all := \
     LICENSE.txt \
     README.md \
     README_PYPI.md \
     requirements.txt \
-    $(wildcard *.py) \
+		pyproject.toml \
     $(wildcard $( package_name)/*.py) \
```

Comment:
Why is pyproject.toml line indented?

Reply:
Because my editor sometimes uses tabs in a Makefile :-)
Fixed it.